sunpy/net/dataretriever/client.py

```diff
@@ -206,15 +206,12 @@ def _makeargs(self, *args, **kwargs):
             elif isinstance(elem, _Range):
                 a_min = elem.min
                 a_max = elem.max
-                if a_min == a_max:
-                    self.map_[elem.__class__.__name__.lower()] = a_min
+                if isinstance(elem, Wavelength):
+                    prefix = 'wave'
                 else:
-                    if isinstance(elem, Wavelength):
-                        prefix = 'wave'
-                    else:
-                        prefix = ''
-                    minmax = namedtuple("minmax", "{0}min {0}max".format(prefix))
-                    self.map_[elem.__class__.__name__.lower()] = minmax(a_min, a_max)
+                    prefix = ''
+                minmax = namedtuple("minmax", "{0}min {0}max".format(prefix))
+                self.map_[elem.__class__.__name__.lower()] = minmax(a_min, a_max)
```

Comment:
These changes are related to what's said on #2314 - I've removed the above if block on l.199 because otherwise single a.Wavelength were set as single Quantity. I don't know whether there's other _Range that is preferred to get only a value if the range min and max are the same.